app/graphql/types/query_type.rb

```diff
@@ -35,10 +38,15 @@ def get_user_activities(id:)
 
     def get_category_activities(id:)
       Category.find_by(name: id)
-    end 
+    end
 
-    def get_activities 
+    def get_activities
       Activity.all
     end
+
+    def get_delights(id:)
+      cat_name = Category.find(id)
+      possibilities = cat_name.activities.sample(5)
```

Comment:
Nice job. Another way to think about formatting this is to just find the Category in this field. Then in the Category Type file define a custom field of possibilities. That method would have the cat.activities.sample line in it and would return the possibilities. The way you have it now is fine but if you ever want to talk about the other set up I'm happy to walk you through it.